### PR TITLE
Dev-docs: instrument view and logging

### PR DESCRIPTION
**Pull request by gvardany**
This is the dev-docs move of 
[Rendering the Instrument View](http://www.mantidproject.org/Rendering_the_Instrument_View)
and [Logging](http://www.mantidproject.org/Logging) combined with [Logging Good Practice](http://www.mantidproject.org/Logging_Good_Practice)

**To test:**
Build the dev-docs, check the pages read and render fine.

Refs #22082 . 

**Release Notes** 

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####

- [ ] Is the code of an acceptable quality?
- [ ] Does the code conform to the [coding standards](http://www.mantidproject.org/Coding_Standards)?
- [ ] Are the unit tests small and test the class in isolation?
- [ ] If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- [ ] Do changes function as described? Add comments below that describe the tests performed?
- [ ] Do the changes handle unexpected situations, e.g. bad input?
- [ ] Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: dev-docs/source/InstrumentViewer.rst [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+========================
+
+.. contents::
+  :local:
+
+Overview
+--------
+
+Mantid visualisation system allows users to view the geometry of an instrument and also select and query individual detectors.
+An instrument is always shown in a context of a workspace of data recorded on this instrument.
```
Comment (DanNixon) [inline dev-docs/source/InstrumentViewer.rst]: `in the context of a workspace of data` seems to read better to me.

File: dev-docs/source/InstrumentViewer.rst [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+The instrument view also allows users to see close-ups of any component.
+
+.. figure:: images/InstrumentWidget.jpg
+  :align: center
+  :width: 635
+
+Implementation
+--------------
+
+MantidPlot uses OpenGL to plot the geometry. Instrument3DWidget represents the view window and controls the viewport and the scene.
```
Comment (DanNixon) [inline dev-docs/source/InstrumentViewer.rst]: Class names should be `inline code formatted`.

File: dev-docs/source/Logging.rst [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
++---------------+-----------------------------------------------------------------------------------------------+
+| FATAL         | An unrecoverable error has occurred and the application will terminate.                       |
++---------------+-----------------------------------------------------------------------------------------------+
+
+Configuring the Log Level
+-------------------------
+
+For the logging to work you will need to have configured the logging service. This will occur when you do either of the following:
+
+  - Call *FrameworkManager.initialise()*
```
Comment (DanNixon) [inline dev-docs/source/Logging.rst]: Use `code/raw text` formatting

File: dev-docs/source/Logging.rst [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+  logging.channels.fileChannel.formatter.pattern = %Y-%m-%d %H:%M:%S,%i [%I] %p %s - %t
+  logging.formatters.f1.class = PatternFormatter
+  logging.formatters.f1.pattern = %s-[%p] %t
+  logging.formatters.f1.times = UTC
+
+This specifies that the logging comments will go to the console as well as a file called *mantid.log*.
+In the example here the level is set to debug, so all the messages will be output.
+In production this will usually be set to information.
+One could also alter the logging level programmatically using *ConfigService*.
+For example, in python:
```
Comment (DanNixon) [inline dev-docs/source/Logging.rst]: "Python"

**Review by DanNixon — CHANGES_REQUESTED** [review]
Looks good. Documentation is clear and reads well.

Only issue is lack of formatting for inline code and filenames which should be as `raw text`.

**Review by DanNixon — APPROVED** [review]

**Comment by martyngigg** [comment]
The failing build is unrelated.

**Pull request merged**